cli/command/swarm/opts.go

```diff
@@ -24,13 +24,17 @@ const (
 	flagToken               = "token"
 	flagTaskHistoryLimit    = "task-history-limit"
 	flagExternalCA          = "external-ca"
+	flagKeepOldSnapshots    = "keep-old-snapshots"
```

Comment:
> Not entirely sure about this flag's name. Name feels like a "boolean" "Yes, keep old snapshots"

It did it this way to be consistent with the API. Maybe `max-old-snapshots`?

> Is there a reason we're specifying old snapshots? Would it make sense to specify the total (including current)?

I guess it would be okay. On the API side it's structured like this so that the 0 value is a sane default. I feel like it might be confusing to have the numbers specified to the CLI be off-by-one from what actually gets sent over the API. It's pretty easy to mess that up in the code, or have a user get the wrong idea.